docs/styling.md

```diff
@@ -0,0 +1,20 @@
+![SeAT](https://i.imgur.com/aPPOxSK.png)
+
+# Styling
+
+By default, SeAT is shipped using Bootstrap 3 and the template Admin LTE.
+
+You may want to customise a bit your own SeAT design to either your corporation or alliance colors.
+
+To doing so, you can use two available css hooks :
+ - `custom-layout-mini.css` which is used by the sign-in page
+ - `custom-layout.css` which is used for all the application
+
+Both of them must be located into your `public` directory.
+
+!!! exemple
```

Comment:
> Example

Reply:
this one is an admonition render :)
https://squidfunk.github.io/mkdocs-material/extensions/admonition/#example

Reply:
Just spell it as example and not exemple.

Reply:
oh, my bad 🙄